lib/master.js

```diff
@@ -195,9 +195,35 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                 type: 'config',
                 body: yaml.dump(self.config)
             });
+
+            var workerExit = function(code) {
```

Comment:
We could actually re-evaluate the purely sequential startup strategy. The main reason for starting up sequentially was to avoid concurrent schema changes. Since only the first worker normally performs those changes, it should be okay to switch to parallel startup after the first worker is up.